### ISSUE TITLE
Client.Timeout results in outputting usage

### ISSUE THREAD

**Issue by karlvr** [sensu/http-checks#13]
We're using these checks in our Sensu cluster. Thank you!

When a site times out we get unexpected output from the check. `http-check` outputs "Usage" information and then an error about the timeout.

I'm hoping / expecting it can output a normal CRITICAL error message.

This is what we're seeing as the output in Sensu when the timeout occurs:

```
Usage:
 http-check [flags]
 http-check [command]
Available Commands:
 help        Help about any command
 version     Print the version number of this plugin
Flags:
 -H, --header strings           Additional header(s) to send in check request
 -h, --help                     help for http-check
 -i, --insecure-skip-verify     Skip TLS certificate verification (not recommended!)
 -C, --mtls-cert-file string    Certificate file for mutual TLS auth in PEM format
 -K, --mtls-key-file string     Key file for mutual TLS auth in PEM format
 -r, --redirect-ok              Allow redirects
 -s, --search-string string     String to search for, if not provided do status check only
 -T, --timeout int              Request timeout in seconds (default 15)
 -t, --trusted-ca-file string   TLS CA certificate bundle in PEM format
 -u, --url string               URL to test (default "http://localhost:80/")
Use "http-check [command] --help" for more information about a command.
Error executing http-check: error executing check: Get https://www.example.com/: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```

(Domain name adjusted for privacy)

I'm not experienced enough with Go to take a crack at this myself. Hoping it's an easy fix.

**Comment by nixwiz** [comment]
Was this addressed in #15 @jspaleta ?

**Comment by jspaleta** [comment]
This should be addressed..
Running a local build from 0.7.0 repository tag seems to work as expected

```
$ nc -l 10001 &
$ ./http-check -u http://localhost:10001 -T 1
request error: Get "http://localhost:10001": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```
In fact 0.5.0+ release should work as well.

Closing
